### PR TITLE
Update docs for 4.18.2.0

### PR DESCRIPTION
**Pull request by JoaoJandre**
Update docs for 4.18.2.0

### CODE REVIEW

**Comment by JoaoJandre** [comment]
@rohityadavcloud @weizhouapache @shwstppr @nvazquez could you please review?

**Comment by JoaoJandre** [comment]
@blueorangutan docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -13,11 +13,29 @@
...
+- Several Hypervisor (VMware, KVM, XenServer) fixes and improvements
+- Several UI fixes and improvements
+- Several Network (L2, VXLAN, etc) fixes and improvements
+- Several System VM (CPVM, SSVM) fixes and improvements
+
+The full list of fixes and improvements can be found in the project release notes at
+https://docs.cloudstack.apache.org/en/4.18.2.0/releasenotes/changes.html
+
+
+
```
Comment (rohityadavcloud) [inline source/releasenotes/about.rst:34]: Minor nit - remove extra spaces?

File: source/releasenotes/changes.rst [hunk 2]
```diff
@@ -14,7 +14,425 @@
...
+===================================
+
+Apache CloudStack uses GitHub https://github.com/apache/cloudstack/milestone/29?closed=1
+to track its issues.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+
++-------------------------+----------+---------------+----------+------------------------------------------------------------+
+| Version                 | Github   | Type          | Priority | Description                                                |
```
Comment (rohityadavcloud) [inline source/releasenotes/changes.rst:27]: Nit - maybe in the generator script (py), we can remove the type and priority columns as they are empty;  for example, you can also use this https://github.com/shapeblue/generate_acs_rn (I had to create the fork as I didn't have push rights to the original one)

**Review by rohityadavcloud — APPROVED** [review]
LGTM - minor concerns raised, good to go. @JoaoJandre let me know if you have access of readthedocs, https://readthedocs.org/accounts/login/ - if not create an account and share your username. Alternative, once ready we can hit publish for you too. cc @DaanHoogland @weizhouapache

File: source/releasenotes/changes.rst (hunk 2 above)
Comment (JoaoJandre) [reply source/releasenotes/changes.rst]: done

**Comment by JoaoJandre** [comment]
@blueorangutan docbuild

**Comment by JoaoJandre** [comment]
> LGTM - minor concerns raised, good to go. @JoaoJandre let me know if you have access of readthedocs, https://readthedocs.org/accounts/login/ - if not create an account and share your username. Alternatively, once this PR and git tag is ready we can hit publish for you too. cc @DaanHoogland @weizhouapache

@rohityadavcloud  I don't have admin access to the readthedocs, could you give me access? my username is: `JoaoJandre`.

**Comment by rohityadavcloud** [comment]
@JoaoJandre done, I've added you as a maintainer/admin on https://readthedocs.org/projects/cloudstack-documentation/ you should get receive an invite. After you log in, the steps to publishing a git tag is easy; (a) merge this PR and create a git push on this repo, (b) go to https://readthedocs.org/projects/cloudstack-documentation/ and under version or admin enable the version (4.18.2.0) and check (c) it should start the build and pass successfully, (d) edit the release/tag and make it public, that's it.

**Comment by rohityadavcloud** [comment]
Cheers @JoaoJandre I see the tag and docs are live now, the website is updated too. I think only thing remaining is the ML announcement for you to draft/send.

**Pull request merged**